portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -39,4 +39,32 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
-}
+
+@Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+    // messagesList.add(text);
+
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("content", text);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(commentEntity);
+
+    // Redirect back to the HTML page.
+    response.sendRedirect("/index.html");
+  }
+
+  /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
+   */
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
`defaultValue` is always set to "" from the caller, so just hardcode it in the method instead of having it as an argument.

Reply:
Done! Thank you